pkg/sessions/cookie/session_store.go

```diff
@@ -129,6 +130,7 @@ func NewCookieSessionStore(opts *options.SessionOptions, cookieOpts *options.Coo
 // it into a slice of cookies which fit within the 4kb cookie limit indexing
 // the cookies from 0
 func splitCookie(c *http.Cookie) []*http.Cookie {
+	logger.Printf("WARNING: Sessions exceeds 4kb cookie limit. Multiple cookies are required for this session. Please use server side session storage (eg. Redis).")
```

Comment:
I think something like this reads better:

```suggestion
	logger.Printf("WARNING: Multiple cookies are required for this session as it exceeds the 4kb cookie limit. Please use server side session storage (eg. Redis) instead")
```